index.mjs

```diff
@@ -30,6 +30,7 @@ import shq from 'shq'
 export function $(pieces, ...args) {
   let __from = (new Error().stack.split('at ')[2]).trim()
   let cmd = pieces[0], i = 0
+  const verbose = $.verbose
```

Comment:
Let’s use let to be consistent with other zx parts.

Reply:
Sure. Fixed.